src/LightGraphsExtras.jl

```diff
@@ -6,7 +6,7 @@ export Datasets
 export Community
 export Interdiction
 
-include("matching/matching.jl")
+include("matching/Matching.jl")
```

Comment:
Keep this lower-case. It causes confusion between case-sensitive and case-insensitive filesystems.